price/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: "2.4"
```

Comment:
Remove? I don't think we have docker compose for any of the other oracles

Reply:
i would remove this, but only because this is already available in the root of the project (we do have a single docker-compose file that stands up the iot-config service, minio with all the right buckets and a postgres database with all the right tables
you _can_ stand up all of the things at once by just typing `docker-compose up -d` but by chaining the depends_on key correctly you can just stand up the one or more services you care about by naming it as an argument to the command e.g. `docker-compose up -d iot-config`

Reply:
Removed in [a454294](https://github.com/helium/oracles/pull/400/commits/a454294be181d1c7e47fa93265e510712be6cdc0)